src/test/shell/integration/bazel_command_log_test.sh

```diff
@@ -45,6 +45,7 @@ function strip_lines_from_bazel_cc() {
     -e '/^WARNING: Waiting for server process to terminate (waited 5 seconds, waiting at most 60)$/d' \
     -e '/^WARNING: The startup option --host_javabase is deprecated; prefer --server_javabase.$/d' \
     -e '/^WARNING: The home directory is not defined, no home_rc will be looked for.$/d' \
+    -e '/Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release/d' \
```

Comment:
Should https://github.com/bazelbuild/bazel/issues/13097 be addressed first, to avoid log spam? I'm fine with just removing `-Xverify:none` at this point

Reply:
Okay—I had the impression that the logspam was preferred over the performance hit from bytecode verification.

I should just delete this line? https://github.com/bazelbuild/bazel/blob/a67c823938dcc426952d25b3a68b825cbe5f6586/src/main/cpp/blaze.cc#L370

Reply:
@benjaminp When exactly do we print the line?